<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BannerWidget</class>
 <widget class="QWidget" name="BannerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="7" column="1">
      <widget class="QSpinBox" name="retentionDurationSpin">
       <property name="toolTip">
        <string/>
       </property>
       <property name="suffix">
        <string>秒</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="fixedWidthSpin">
       <property name="toolTip">
        <string/>
       </property>
       <property name="suffix">
        <string>像素</string>
       </property>
       <property name="prefix">
        <string>固定</string>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="frostedGlassOpacitySpin">
       <property name="toolTip">
        <string/>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string>不透明度</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>32</number>
       </property>
       <property name="value">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="thumbnailPropSpin">
       <property name="toolTip">
        <string/>
       </property>
       <property name="suffix">
        <string>分之一</string>
       </property>
       <property name="prefix">
        <string>最大</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="toolTip">
        <string>在弹窗中图片最大显示的尺寸
如果图片过大，将会压缩得更小</string>
       </property>
       <property name="text">
        <string>缩略图显示比例</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="colorfulGroupMemberCheck">
       <property name="text">
        <string>群组成员彩色昵称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="displayDurationSpin">
       <property name="suffix">
        <string>秒</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>通知卡片出现的位置</string>
       </property>
       <property name="text">
        <string>横幅出现位置</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="useHeaderGradientCheck">
       <property name="text">
        <string>使用头像渐变</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="textReadSpeedSpin">
       <property name="toolTip">
        <string/>
       </property>
       <property name="suffix">
        <string>字</string>
       </property>
       <property name="prefix">
        <string>每秒</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="contentMaxHeightSpin">
       <property name="toolTip">
        <string/>
       </property>
       <property name="suffix">
        <string>像素</string>
       </property>
       <property name="prefix">
        <string>最大</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>横幅显示时长</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="InteractiveButtonBase" name="bgColorButton">
       <property name="text">
        <string>背景颜色</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="frostedGlassBgCheck">
       <property name="toolTip">
        <string>使用简单的毛玻璃背景效果
注意：仅在出现时静态模糊，不会随着底部内容的修改而变化</string>
       </property>
       <property name="text">
        <string>毛玻璃背景</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="privateKeepShowingCheck">
       <property name="toolTip">
        <string>私聊的通知出现后，不会自动隐藏（相当于未读）
除非用户操作后超时关闭，例如回复、鼠标移入再移开</string>
       </property>
       <property name="text">
        <string>私聊通知保持显示</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="groupKeepShowingCheck">
       <property name="toolTip">
        <string>群聊的通知出现后，不会自动隐藏（相当于未读）
除非用户操作后超时关闭，例如回复、鼠标移入再移开</string>
       </property>
       <property name="text">
        <string>群聊通知保持显示</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="floatPixelSpin">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;距离四边的距离&lt;/p&gt;&lt;p&gt;右上/左上：距离顶边&lt;/p&gt;&lt;p&gt;右下/左下：距离底边&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="suffix">
        <string>像素</string>
       </property>
       <property name="prefix">
        <string>距边缘</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>失去焦点（鼠标移开/回复框关闭）后多久时间隐藏
一个小技巧：慢慢移开可快速隐藏通知，快速移开则可以继续保留</string>
       </property>
       <property name="text">
        <string>鼠标移开后再停留</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="toolTip">
        <string>通知卡片出现的位置</string>
       </property>
       <property name="text">
        <string>横幅内容高度</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string>通知卡片出现的位置</string>
       </property>
       <property name="text">
        <string>横幅显示宽度</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QCheckBox" name="useHeaderColorCheck">
       <property name="text">
        <string>跟随头像变色</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>根据内容的长度，自动调整横幅通知显示的时长</string>
       </property>
       <property name="text">
        <string>文字阅读速度</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="InteractiveButtonBase" name="titleColorButton">
       <property name="text">
        <string>文字颜色</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>横幅显示时长</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="positionCombo">
       <item>
        <property name="text">
         <string>右上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>右下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>左上</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>左下</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>127</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButtonBase</class>
   <extends>QPushButton</extends>
   <header>interactivebuttonbase.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
